src/algebra/algebra/basic.lean

```diff
@@ -1195,6 +1199,51 @@ by { ext, simp only [linear_map.id_coe, mul_one, id.def, lmul_right_apply] }
   lmul_right R (a * b) = (lmul_right R b).comp (lmul_right R a) :=
 by { ext, simp only [lmul_right_apply, linear_map.comp_apply, mul_assoc] }
 
+@[simp] lemma lmul_left_zero_eq_zero :
+  lmul_left R (0 : A) = 0 :=
+by { ext, simp, }
```

Comment:
Not shorter, but gives a stronger hint "we already proved this"
```suggestion
(lmul R A).map_zero
```

Reply:
Thanks!